supernode/util/range_util.go

```diff
@@ -26,6 +26,30 @@ const (
 	separator = "-"
 )
 
+// CalculatePieceSize calculates the size of piece
+// according to the parameter range.
+func CalculatePieceSize(rangeStr string) int64 {
```

Comment:
Could you add unit test case for this function? Thanks.
I think everyone should keep in mind that test case is the good way to guarantee the software quality. ❤️

Reply:
Thanks for pointing out this. Updated!

Reply:
I think you can compress `CalculatePieceSize` and `CalculatePieceNum` methods.

Reply:
What do you mean by `compress` this two methods? Do you mean use one single method to return this two values?